scripts/nightly_smoke_ui_tests.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Can this be a Jenkins file? cc @Victoremepunto

Reply:
I believe you could add a condition for this to run on schedule against a fixed branch, with a cron schedule directly on the pipeline. I have not tested this myself. This pipeline is mimicking what Advisor does for their smoke test builds . 

Similarly checking FE availability I must say, perhaps this pipeline is not required if relying on Advisor's smoke test pipeline is enough ? This will not cover Compliance-specific health checks, of course, but checking the a FE deployment "works" should be covered - where "works" means: we are able to deploy a CRC aaplication on EE and we are able to successfully navigate to one of the views and interact with parts of if through a Selenium browser (effectively validating a FE application is able to run on EE).

Reply:
So what I hear is that a parametrization in a Jenkins file and a regular schedules for a specific branch would do it.

Reply:
Yes that is correct. I have not tried this myself (on our current app-interface context, that is, it does work on "vanilla" Jenkins) but that should do the trick. 

@LightOfHeaven1994  LMK if you need help testing this ^

Reply:
@Victoremepunto I will, let's speak about this on Friday meeting if possible. Thanks!